docs/change_log/2022_09_automod_spam.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: AutoMod Spam and Mention Spam Trigger Types
+date: 2022-09-21
+---
+
+Two new [trigger types](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-trigger-types) were added to AutoMod:
+
+- `MENTION_SPAM` blocks messages that mention more than a set number of unique server members or roles. Apps can define the number (up to 50) using the `mention_total_limit` field in the [trigger metadata object](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object-trigger-metadata) when creating or updating an Auto Moderation rule.
+- `SPAM` blocks links and messages that are identified as spam.
+
+More information can be found in the [AutoMod documentation](#DOCS_RESOURCES_AUTO_MODERATION).
```

Comment:
I don't think "AutoMod" should be used here. See https://github.com/discord/discord-api-docs/pull/4860#discussion_r876503657 - the other Auto Moderation change log respects that comment.